### PR TITLE
basic webpack

### PR DESCRIPTION
**Pull request by dradetsky**
probably don't want to straight up merge this, but it might be a useful starting point

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        loader: 'babel-loader',
+        options: {
+          presets: ['react', 'es2015']
+        }
+      }
+    }]
+  }
+}
+
+module.exports = [config]
```
Comment (elimisteve) [inline webpack.config.js]: @dradetsky `config` should be in an array for some reason?

Comment (dradetsky) [inline webpack.config.js]: honestly, i can't remember why that's done

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+const config = {
+  entry: {
+    vendor: './src/vendor.js',
+    main: './src/index.js',
+  },
+  output: {
+    path: path.join(__dirname, 'build'),
+    publicPath: '/build',
+    filename: '[name].[chunkhash].js',
+    chunkFilename: '[name].[chunkhash].js'
```
Comment (elimisteve) [inline webpack.config.js]: @Spetastian Seems like we need ^^this^^ for cache-busting?

**Comment by elimisteve** [comment]
Thanks for this!  We used part of this for the current config :+1: which is working fantastically.

Thanks, gents!

**Pull request closed without merging**